README.md

```diff
@@ -148,7 +169,16 @@ Write ruby code that will create a person.
 
 Your answer:
 ```
-Replace this with your answer
+  class Person
+  attr_accessor :age, :name
+  @@all = []
+
+  def initialize age, name
+    @age = age
+    @name = name
+    @@all.push(self)
+  end
+
 ```
```

Comment:
this question is assuming already having defined a Person class with the given schema. So we would just write:

```ruby
Persone.create(name: "bob", age: 15)
```